Assets/Scripts/Cards/Card.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public abstract class Card : ScriptableObject {
```

Comment:
This class doesn't contain any abstract members so it shouldn't be marked `abstract`